cfy_manager/components/nginx/config/manager.upstream

```diff
@@ -1,3 +1,4 @@
+{% macro lo() %}{% if ipv6_enabled %}[::]{% else %}127.0.0.1{% endif %}{% endmacro %}
```

Comment:
The equivalent of :: is 0.0.0.0 I think?

Reply:
Or, if you wanted local, then ::1 is the IPv6 one

Reply:
right, thanks